assignment.js

```diff
@@ -32,6 +32,13 @@ console.log('Problem 1:')
 
 // Add your code below this line
 
+const n=2
+console.log("Base number: " + `${n}`)
+console.log("Result:")
+for (let i = 0; i <= 12; i++) {
+  console.log(i*n)
+}
+
```

Comment:
@AtleLohrmann Good job here implement the for loop to loop through each value up to 12 and multiply those values by 2. Good job on the indentation here
Take a look at this approach [here](https://github.com/jsr20190826/hw-01-js-basics-control-flow-loops-solution/blob/a810fe3f581584e5015519a3508b078d38c4c8c9/assignment.js#L31)